pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -38,15 +39,24 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/component-helpers/storage/ephemeral"
+	runtimeapi "k8s.io/cri-api/pkg/apis/runtime/v1"
 	"k8s.io/kubernetes/pkg/kubelet/config"
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
+	"k8s.io/kubernetes/pkg/kubelet/metrics"
 	"k8s.io/kubernetes/pkg/kubelet/volumemanager/cache"
 	"k8s.io/kubernetes/pkg/volume"
 	"k8s.io/kubernetes/pkg/volume/csimigration"
 	"k8s.io/kubernetes/pkg/volume/util"
 	volumetypes "k8s.io/kubernetes/pkg/volume/util/types"
 )
 
+const (
+	dswpEventChannelCapacity                    = 1000
+	dswpEventMaxStreamRetries                   = 5
+	dswpEventMinLoopBeforeIncreasingSleepPeriod = 3
+	initialLoopSleepDuration                    = time.Duration(100) * time.Millisecond
```

Comment:
could be re-written as - `100 * time.Millisecond` ?

Reply:
updated